xwiki-platform-core/xwiki-platform-oldcore/src/test/java/com/xpn/xwiki/doc/XWikiDocumentMockitoTest.java

```diff
@@ -1698,7 +1698,7 @@ void tofromXMLDocument() throws XWikiException
         this.baseObject.removeField("passwd");
         this.baseObject2.removeField("passwd");
         // We also need to remove the field from the sourceXClass, else it will appear as an empty field value.
-        this.baseObject.getSourceXClass().removeField("passwd");
+        //this.baseObject.getSourceXClass().removeField("passwd");
```

Comment:
This line should be removed together with the comment above it if it isn't needed anymore.